composer.json

```diff
@@ -43,7 +43,7 @@
         "drupal/piwik": "~1.0",
         "drupal/piwik_reporting_api": "^1.0-beta2",
         "drupal/r4032login": "dev-1.x#391f10d69b9b8c466a2fddac2a6b99f1289e7781",
-        "drupal/rdf_entity": "dev-8.x-1.x",
+        "drupal/rdf_entity": "dev-graph-store-trait",
```

Comment:
I do not see a PR on the upstream repo or any ticket related to that branch. How are we going to keep that here?

Reply:
Yepp, I forgot to add the PR. Here it is: https://github.com/ec-europa/rdf_entity/pull/22